src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingScriptOperation.cs

```diff
@@ -280,23 +305,24 @@ private void OnPublishModelScriptError(object sender, ScriptEventArgs e)
             Logger.Write(
                 LogLevel.Verbose,
                 string.Format(
-                    "Sending scripting error progress event, Urn={0}, OperationId={1}, Completed={2}, Error={3}",
+                    "Sending scripting error progress event, Urn={0}, OperationId={1}, Sequence={2}, Completed={3}, Error={4}",
                     e.Urn,
                     this.OperationId,
+                    this.eventSequenceNumber,
                     e.Completed,
-                    e.Error));
+                    e?.Error?.ToString()));
```

Comment:
It works since I' using more than 3 arguments which invokes a particular overload of string.format. I'll update to use the null coalescing operator to make this more clear. 

---
In reply to: [115635528](https://github.com/Microsoft/sqltoolsservice/pull/337#discussion_r115635528) [](ancestors = 115635528)